android-lib/src/androidTest/java/com/kinvey/androidTest/store/UserStoreTest.java

```diff
@@ -374,6 +374,30 @@ public void testLogin() throws InterruptedException {
         assertNull(logout(client).error);
     }
 
+    @Test
+    public void get() throws InterruptedException {
+        login(USERNAME, PASSWORD);
+        DefaultKinveyClientCallback callback = get(client.getActiveUser().getId());
+        assertNull(callback.error);
+        assertNotNull(callback.result);
+    }
```

Comment:
In this test, we should also verify that we get back the expected `User` object, in addition to checking for a non-null result.